modules/apps/object/object-test/src/testFunctional/paths/ObjectAction.path

```diff
@@ -22,6 +22,36 @@
 		<td>//button[contains(.,'${key_tabName}')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>INPUT_VALUE</td>
+		<td>//input[(@placeholder='Input a value.')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>INPUT_EXPRESSION</td>
+		<td>//input[(@placeholder="Create an expression.")]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>INPUT_NEW_VALUE</td>
+		<td>//input[(@placeholder="Input a value or create an expression.")]</td>
+		<td></td>
+	</tr>
```

Comment:
I'm not going to enforce this for now but we could create a generic input path instead and pass in the different texts to the path.